playbooks/openshift/README.md

```diff
@@ -44,14 +44,16 @@ This is a log of an example installation of a proof of concept cluster with
 Shell environment with
 - OpenStack credentials for cPouta 
 - python virtualenv with ansible==2.3, shade, dnspython and pyopenssl
-- venv should have latest setuptools and pip (pip install --upgrade setuptools pip)
+- venv should have latest setuptools and pip (pip install --upgrade pip setuptools)
 - metrics needs java keytool on the bastion host: sudo yum install java-1.8.0-openjdk-headless
 - if you have SELinux enabled, either disable that or make sure the virtualenv has libselinux-python  
 - ssh access to the internal network of your project
     - either run this on your bastion host
     - or set up ssh forwarding through your bastion host in your ~/.ssh/config
     - please test ssh manually after provisioning 
 
+For packages on CentOS-7, see: [Creating a bastion host](../../CREATE_BASTION_HOST.md)
```

Comment:
Maybe the change the CREATE_BASTION_HOST.md to use the new provisioning code. I'd keep that out of the scope of this PR